twitter-export-image-fill.py

```diff
@@ -58,6 +58,24 @@
 
 image_count_global = 0
 
+# Function to download an avatar (from tweet or retweet)
+
+def download_avatar(user):
+  avatar_url = user['profile_image_url_https']
+  extension = os.path.splitext(avatar_url)[1]
+  screen_name = user['screen_name']
+  local_filename = "img/avatars/%s%s" % (screen_name, extension)
+  downloaded = 0
+  if not os.path.isfile(local_filename):
+    urllib.urlretrieve(avatar_url, local_filename)
+    downloaded = 1
+  user['profile_image_url_https_orig'] = avatar_url
+  user['profile_image_url_https'] = local_filename
+  return downloaded
+
+if not os.path.isdir("img/avatars"):
```

Comment:
This is a different way to check for a presence of a directory than I used. Can you either use mine, or switch mine (os.stat) to yours? Also, add a comment, please.